swagger.yml

```diff
@@ -225,10 +225,10 @@ definitions:
   setup:
     type: object
     properties:
-      display_name:
+      username:
         type: string
```

Comment:
I would add description here with an example - Should this be an email or any plaintext string work?